esp/src/src/nls/es/hpcc.ts

```diff
@@ -338,8 +343,15 @@ export = {
     FromTime: "Tiempo de Comenzar",
     FullName: "Nombre Entero",
     Generate: "Generar",
+    GetDFSCSV: "CSV de DFS",
```

Comment:
L346-350 seen to be missing "Obtener"

Reply:
The actual phrases for the get calls in the english file do not include get as it was not required, but the tag for it retained the get. The English version does not include Get, in the tab names on ECL Watch for reference.

Reply:
fair enough, that's why I keep calling for these translation to provide the actual English text for context. 
With that said, the phrases on their own don't make much sense, but do not seem to be incorrect.

Reply:
Retained translations as is without including get to conform with the english version.